app/models/produce_item.rb

```diff
@@ -8,4 +8,8 @@ class ProduceItem < ApplicationRecord
 
   validates :name, presence: true
   validates :category, presence: true
+
+  def name_with_whitespace
+    name.gsub(/_/, ' ')
```

Comment:
I think you do not need to use a regular expression here. But please check if that's the case.

Reply:
we can use `name.gsub('_', ' ')`, is there an advantage?

Reply:
Simple string replacement has better performance than regular expressions :D